src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -321,7 +330,7 @@ public int compare( GenericRecord o1, GenericRecord o2 ) {
 
         for (final GenericRecord queryRow : avroReader) {
             long location = (Long) queryRow.get(SchemaUtils.LOCATION_FIELD_NAME);
-            int length = Integer.parseInt(queryRow.get(SchemaUtils.LENGTH_FIELD_NAME).toString());
+            int length = ((Long) queryRow.get(SchemaUtils.LENGTH_FIELD_NAME)).intValue();
```

Comment:
(for my edu only) Is it better to not convert to a string in the first place?

Reply:
it's expensive… to convert a number to a string and then parse the string to get back another number.  the result from `get` is already a Long we just have to cast it as such.  BigQuery doesn't return int, but we know it is an int and want it as such so we call `intValue()` on it.